javascript/javascript1/week2/weather_wear.js

```diff
@@ -0,0 +1,21 @@
+function smartWear(temp) {
+  const clothes = [
+    "shorts and a t-shirt",
+    "jeans and a sweater",
+    "light coat",
+    "winter coat and a winter hat",
+  ];
+  if (temp >= 18) {
+    return clothes[0];
+  } else if (temp >= 12) {
+    return clothes[1];
+  } else if (temp >= 5) {
+    return clothes[2];
+  } else if (temp >= -5) {
+    return clothes[3];
+  } else {
+    return "Stay home! It's freezing!";
+  }
+}
```

Comment:
Nice! Can you think of a nested array and for-looping over temps and clothes synchronously, with a single if statement inside the for-loop?

Reply:
Yes! Sweet!